src/core/scene/a-scene.js

```diff
@@ -301,6 +301,19 @@ class AScene extends AEntity {
               vrManager.setSession(xrSession).then(function () {
                 vrManager.setFoveation(rendererSystem.foveationLevel);
               });
+              const rates = xrSession.supportedFrameRates;
+              if (rates && xrSession.updateTargetFrameRate) {
+                let targetRate;
+                if (rates.includes(90)) {
```

Comment:
Usually I prefer to avoid "mmagic numbers" in code, but I'm not sure that replacing these numbers with constants would do anything to aid maintenance or readability.

If you'd like me to use constants like FRAME_RATE_90 etc., please point me to where you'd like them defined.  Top of this module?  constants/index.js?  Or somewhere else?

Reply:
can this be factored out to a function? `setImmersiveModeRefreshRate`. May be the code should be part of the renderer component?

Reply:
OK, good suggestion - I made this a function on the renderer component & added unit tets for that